CIPs/0011.md

```diff
@@ -0,0 +1,123 @@
+# CIP 11: Off-chain claims and attestations
+
+- Date: 2020-05-20
+- Author: @nambrot
```

Comment:
Hi @nambrot as the other PRs, please ensure this CIP follows the new CIP template in the newly activated CIP process as found here https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-template.md